vo-dml/desc.mango.vo-dml.xml

```diff
@@ -883,54 +869,54 @@ providers to convey whether or not the factor has been applied (see \texttt{meas
     <vodml-id>EpochPositionErrors</vodml-id>
     <name>EpochPositionErrors</name>
     <description>Class holder for the errors of the EpochPosition attributes</description>
-    <composition>
+    <attribute>
       <vodml-id>EpochPositionErrors.parallax</vodml-id>
       <name>parallax</name>
       <description> Parallax error. This error is meant to be symmetrical in the current model version.</description>
       <datatype>
-        <vodml-ref>mango:error.Symmetric1D</vodml-ref>
+        <vodml-ref>mango:error.PropertyError</vodml-ref>
       </datatype>
       <multiplicity>
-        <minOccurs>0</minOccurs>
+        <minOccurs>1</minOccurs>
         <maxOccurs>1</maxOccurs>
       </multiplicity>
-    </composition>
-    <composition>
+    </attribute>
+    <attribute>
       <vodml-id>EpochPositionErrors.radialVelocity</vodml-id>
       <name>radialVelocity</name>
       <description>Error in the radial velocity.  This error is meant to be symmetrical in the current model version.</description>
       <datatype>
-        <vodml-ref>mango:error.Symmetric1D</vodml-ref>
+        <vodml-ref>mango:error.PropertyError</vodml-ref>
       </datatype>
       <multiplicity>
-        <minOccurs>0</minOccurs>
+        <minOccurs>1</minOccurs>
         <maxOccurs>1</maxOccurs>
       </multiplicity>
-    </composition>
-    <composition>
-      <vodml-id>EpochPositionErrors.position</vodml-id>
-      <name>position</name>
+    </attribute>
+    <attribute>
+      <vodml-id>EpochPositionErrors.properMotion</vodml-id>
+      <name>properMotion</name>
       <description> Position error; can be an ellipse, a correlation matrix or a covariance matrix.</description>
```

Comment:
properMotion and position are both described with the same text as "Position error", is this intentional?